src/flp/types/fixedpoint_l2.rs

```diff
@@ -323,21 +346,70 @@ where
             gadget1_chunk_len,
         })
     }
+
+    /// This noising function can be called on the aggregation result to make
```

Comment:
Also, don't we intend this to be called on aggregate shares?
```suggestion
    /// This noising function can be called on the aggregate result to make
```

Reply:
Yes, I changed it to "aggregate share".